src/pulselistener/pulselistener/listener.py

```diff
@@ -112,10 +112,9 @@ def wait_build():
             logger.info('Published public build as working', build=str(build))
 
         elif build.state == PhabricatorBuildState.Secured:
-            # Report secured bug as failing
-            # and remove it from queue
-            self.api.update_build_target(build.target_phid, BuildState.Fail)
-            logger.info('Published secure build as failing', build=str(build))
+            # We cannot send any update on a Secured build
+            # as the bot has no edit access on it
+            logger.info('Secured revision, skipping.', build=build.target_phid)
```

Comment:
So, what is the UI like on a secure revision? Does the build show as pending?

Reply:
I'm not sure, i think so...

Reply:
Mmmh, not great, we should figure out an alternative.

Reply:
Maybe let's talk about this in the next meeting with engflow.